package.json

```diff
@@ -0,0 +1,112 @@
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Maosorio92/workshop-api-testing-js.git"
+  },
+  "keywords": [
+    "api-testing",
+    "dojo",
+    "practice"
+  ],
+  "author": "[Manuel Alejandro Osorio Vásquez] <[maaosoriova@gmail.com]> ([https://github.com/Maosorio92])",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Maosorio92/workshop-api-testing-js/issues"
+  },
+  "homepage": "https://github.com/Maosorio92/workshop-api-testing-js#readme",
+  "devDependencies": {
+    "chai": "^4.3.6",
+    "mocha": "^10.0.0"
+  },
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "ansi-colors": "^4.1.1",
+    "ansi-styles": "^4.3.0",
+    "anymatch": "^3.1.2",
+    "ansi-regex": "^5.0.1",
+    "argparse": "^2.0.1",
+    "assertion-error": "^1.1.0",
+    "brace-expansion": "^2.0.1",
+    "braces": "^3.0.2",
+    "balanced-match": "^1.0.2",
+    "browser-stdout": "^1.3.1",
+    "camelcase": "^6.3.0",
+    "chalk": "^4.1.2",
+    "binary-extensions": "^2.2.0",
+    "check-error": "^1.0.2",
+    "chokidar": "^3.5.3",
+    "cliui": "^7.0.4",
+    "color-convert": "^2.0.1",
+    "concat-map": "^0.0.1",
+    "color-name": "^1.1.4",
+    "debug": "^4.3.4",
+    "deep-eql": "^3.0.1",
+    "decamelize": "^4.0.0",
+    "diff": "^5.0.0",
+    "escape-string-regexp": "^4.0.0",
+    "emoji-regex": "^8.0.0",
+    "escalade": "^3.1.1",
+    "fill-range": "^7.0.1",
+    "find-up": "^5.0.0",
+    "fs.realpath": "^1.0.0",
+    "get-caller-file": "^2.0.5",
+    "get-func-name": "^2.0.0",
+    "glob": "^7.2.0",
+    "flat": "^5.0.2",
+    "glob-parent": "^5.1.2",
+    "has-flag": "^4.0.0",
+    "he": "^1.2.0",
+    "inflight": "^1.0.6",
+    "inherits": "^2.0.4",
+    "is-binary-path": "^2.1.0",
+    "is-extglob": "^2.1.1",
+    "is-fullwidth-code-point": "^3.0.0",
+    "is-glob": "^4.0.3",
+    "is-number": "^7.0.0",
+    "is-plain-obj": "^2.1.0",
+    "is-unicode-supported": "^0.1.0",
+    "js-yaml": "^4.1.0",
+    "log-symbols": "^4.1.0",
+    "locate-path": "^6.0.0",
+    "ms": "^2.1.3",
+    "loupe": "^2.3.4",
+    "minimatch": "^5.0.1",
+    "nanoid": "^3.3.3",
+    "normalize-path": "^3.0.0",
+    "once": "^1.4.0",
+    "p-limit": "^3.1.0",
+    "p-locate": "^5.0.0",
+    "pathval": "^1.1.1",
+    "picomatch": "^2.3.1",
+    "randombytes": "^2.1.0",
+    "readdirp": "^3.6.0",
+    "path-is-absolute": "^1.0.1",
+    "require-directory": "^2.1.1",
+    "safe-buffer": "^5.2.1",
+    "serialize-javascript": "^6.0.0",
+    "path-exists": "^4.0.0",
+    "string-width": "^4.2.3",
+    "strip-ansi": "^6.0.1",
+    "strip-json-comments": "^3.1.1",
+    "supports-color": "^8.1.1",
+    "to-regex-range": "^5.0.1",
+    "type-detect": "^4.0.8",
+    "workerpool": "^6.2.1",
+    "wrap-ansi": "^7.0.0",
+    "wrappy": "^1.0.2",
+    "yargs": "^16.2.0",
+    "y18n": "^5.0.8",
+    "yargs-parser": "^20.2.4",
+    "yargs-unparser": "^2.0.0",
+    "yocto-queue": "^0.1.0"
+  }
```

Comment:
are these dependencies required? I think that just mocha and chai as dev dependencies are required for this step